### ISSUE TITLE
C-c C-a not being bound

### ISSUE THREAD

**Issue by asmeurer** [Malabarba/latex-extra#27]
For me, C-c C-a is still bound to TeX-command-run-all. I saw in the source that it doesn't bind it if it thinks TeX-command-run-all is already defined. However, I am not defining it anywhere as far as I can tell (it isn't mentioned anywhere in my [`.emacs`](https://github.com/asmeurer/dotfiles/blob/master/.emacs)).  Is it possible I have something else that is defining it? I grepped my `~/.emacs.d` for it and nothing came up. It seems like it's being defined by the auctex mode by default.

**Comment by Knusper** [comment]
Yes - `C-c C-a` is defined by default in more recent versions of `auctex`.  If you want to use the `C-c C-a` binding of here, you need to unbind `C-c C-a` before loading this package.
